src/ui/deleteAccount/ViewDeleteAccountRequest.qml

```diff
@@ -59,9 +61,10 @@ MZInAppAuthenticationBase {
                 .arg("<b style='color:" + MZTheme.theme.fontColorDark + ";'>"
                     + MZAuthInApp.emailAddress + "</b>")
             textFormat: Text.RichText
+            font.pixelSize: MZTheme.theme.fontSize
 
             Layout.fillWidth: true
-            Layout.bottomMargin: MZTheme.theme.vSpacing
+            Layout.bottomMargin: MZTheme.theme.vSpacingSmall
```

Comment:
mocks show 24px of spacing between the subtitle and the checkbox section. here we have the `bottomMargin` (16px) combined with the `spacing` (16px) totaling 32px of spacing

Reply:
Thanks, good catch. Updated.